plugins/plugin-site/package.json

```diff
@@ -58,6 +58,8 @@
   "homepage": "https://plugins.jenkins.io/",
   "dependencies": {
     "@babel/core": "7.26.0",
+    "@emotion/react": "^11.14.0",
+    "@emotion/styled": "^11.14.0",
```

Comment:
Are these two used anywhere?

Reply:
Yes, for the componenets of the component library MUI in new feature

Reply:
Removed with component library